dotnet/src/Connectors/Connectors.MistralAI/MistralAIPromptExecutionSettings.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI;
+
+/// <summary>
+/// Mistral Execution Settings.
+/// </summary>
+public sealed class MistralAIPromptExecutionSettings : PromptExecutionSettings
```

Comment:
```suggestion
[JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
public sealed class MistralAIPromptExecutionSettings : PromptExecutionSettings
```
nit; 😄 from OpenAIPromptExecutionSettings